docs/contributing/999-rfcs/2024-03-31-wing-secrets-cli.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "#6105 Wing Secrets CLI"
+description: Creating Wing secrets from the CLI
+---
+
+# Wing Secrets CLI
+- **Author(s)**: @hasanaburayyan
+- **Submission Date**: 2024-03-31
+- **Stage**: Draft
+
+Creating secrets through the Wing CLI.
+
+## Background
+
+TODO
+
+## Sketch
+
+Running the blow command will result in an interactive prompt where you can input the Slack credentials:
+
+```bash
+wing secrets main.w --create
```

Comment:
```suggestion
wing secrets main.w
```

Why do we need the `--create` switch here?

Reply:
My initial thought was that the default behavior of `wing secrets main.w` would list out required secrets. But default create is probably also fine.

Reply:
As a design principle I usually prefer to optimize for the common use case, which is to set secret values I believe.